js/src/forum/components/UserPage.js

```diff
@@ -88,6 +88,8 @@ export default class UserPage extends Page {
   loadUser(username) {
     const lowercaseUsername = username.toLowerCase();
 
+    app.preloadedApiDocument();
```

Comment:
Why are we not using the result of this method?

Because we only want to support client-side navigation and can not rely on the user getting here directly from a server-rendered page?

Reply:
The result of this method simply returns the `apiDocument` passed through by the server.

I'm not using it as calling `preloadedApiDocument` loads the objects and relationships into the global `app.store`, so the `.find` code below can find new users. I tried using the object it returned, but it was just JSON and not model classes, so the rest of the code wouldn't work with it.

> Because we only want to support client-side navigation and can not rely on the user getting here directly from a server-rendered page?
It returns `null` if it has already been called. But that doesn't matter in this case because of my explanation above.

Reply:
Ah, the method name is confusing in that case, but I don't yet have a better name yet.

We definitely want a comment here, though, to explain that we are relying on this side effect to happen.

Reply:
@franzliedke Will do, no problem.

Reply:
Comment added.